game/scripts/npc/abilities/slardar_bash_oaa.txt

```diff
@@ -26,7 +26,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "5.0"
+    "AbilityCooldown"                                     "4.7"
```

Comment:
slardar with some small love but otherwise a change that doesn't make a significant change. with octarine it's a perfect 3.5 sec cd with a 1.5 sec stun duration leaves us with a perfect 2 second downtime, otherwise slardar players will appreciate the small change nonetheless.

Reply:
Arcane Blink exists and does what you described: ''perfect 3.5 sec cd with a 1.5 sec stun duration leaves us with a perfect 2 second downtime,''. So NO.

Reply:
4.7 looks wrong